### ISSUE TITLE
Get off Cloudgov-cli fork

### ISSUE THREAD

**Issue by jbrown-xentity** [GSA/data.gov#4265]
We recently had to get off https://github.com/cloud-gov/cg-cli-tools upstream, due to failing to download the key (details on cloudfoundry [slack](https://cloudfoundry.slack.com/archives/C032824SM/p1679947509129869) and [github](https://github.com/cloudfoundry/cli/issues/2390)). We discussed internally, and Nick created a workaround: https://github.com/GSA/cg-cli-tools/tree/key-not-accessible. We implemented this for catalog, inventory, and other places. We will need to roll this back, once an upstream fix is in place...

## How to reproduce

1. Can verify if normal process will work by running `wget https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key`

## Expected behavior

- File is downloaded

## Actual behavior

- 403 error

## Sketch

Hopefully this is just a reversal of our PR's. If this lingers, we may have to consider other more stable solutions.

- https://github.com/GSA/catalog.data.gov/pull/901
- https://github.com/GSA/data.gov/pull/4263

**Comment by nickumia-reisys** [comment]
Tested locally and the problem has been fixed.  With the reversion of the two PRs, this should be complete.

See https://github.com/cloudfoundry/cli/issues/2390#issuecomment-1487670880

**Comment by nickumia-reisys** [comment]
References for future work if any:
- https://stackoverflow.com/questions/21338721/gpg-no-valid-openpgp-data-found
- https://stackoverflow.com/questions/68992799/warning-apt-key-is-deprecated-manage-keyring-files-in-trusted-gpg-d-instead
- https://wiki.debian.org/DebianRepository/UseThirdParty#OpenPGP_Key_distribution
- https://opensource.com/article/22/9/deprecated-linux-apt-key
